doc/xml/daps_user_layout.xml

```diff
@@ -176,12 +176,107 @@ XSLTPARAM="$XSLTPARAM --param&nbsp;<replaceable>PARAM_NAME</replaceable>=<replac
  
  <sect1 id="sec.daps.user.layout.styleheets">
   <title>Customizing the DocBook Stylesheets</title>
+  <para>
+    This section describe how you can use customs stylesheets.  There are many
+    reasons why it would be preferable to create your own customization layer.
+    Perhaps you want to change the layout or style of your project, change the
+    behaviour how an element is processed or want to create a corporate style.
+  </para>
+  <important>
+    <para>
+      The reader should be aware of that he or she needs some knowledge about
+      XSLT (at least a little bit). Basic customizations like parameter changes
+      can be done by simple adding a line with:
+      <screen>&lt;xsl:param name="PARAMETER_NAME" select="VALUE"/></screen>
+      The available parameters are format dependent and are listed in the
+      "XSL Stylesheets Reference Documentation".
```

Comment:
Add a link to `http://docbook.sourceforge.net/release/xsl/current/doc/param.html` for the "XSL Stylesheet Reference Documentation".